release/templates/featuresenabled-azuredeploy.json

```diff
@@ -74,6 +74,7 @@
             "DicomFunctions__ConnectionName": "AzureWebJobsStorage",
             "DicomServer__Security__Enabled": "true",
             "DicomServer__Security__Authorization__Enabled": "true",
+            "DicomServer__Features__SupportNewBlobFormatForNewService": "true",
```

Comment:
Suggestion - simplify to `SupportNewBlobFormat`

Reply:
Changed to enum